aws/aws.go

```diff
@@ -0,0 +1,21 @@
+package aws
+
+// NS -
+func NS() *awsFuncs {
+	if af == nil {
```

Comment:
This conditional should be replaced with a call to [`sync.Once`](https://golang.org/pkg/sync/#Once) or initialized in an [`init()`](https://github.com/hashicorp/go-sockaddr/blob/master/template/template.go#L27-L102) function.  `sync.Once` has the advantage of you only pay for what you use, but `init()` is a bit more simple to grok and idiomatic.